app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,7 @@
         <activity
             android:name=".ui.MainActivity"
             android:exported="false"
+            android:noHistory="true"
```

Comment:
Don't use this. This is a hacky workaround and you will not gain understanding of backstacks and activity stacks.